src/components/WalletUnlock/DeterministicAccountTable.tsx

```diff
@@ -348,8 +348,8 @@ const DeterministicTable = ({
                 </DPathContainer>
                 <ValueContainer>
                   <Typography>
-                    <Amount
-                      assetValue={
+                    <Currency
```

Comment:
Why do we prefer `Currency` over `Amount` for some things?

Reply:
`Currency` is just a float with an ticker and an optional Icon. 
`Amount` includes the display of the value converted to a `baseAsset` and or a `Fiat`.

In this case, since we only display need a `Currency` I modified to use the correct component.